modules/test/playwright/tests/frontend-data-set-views-web/dataSets.spec.ts

```diff
@@ -0,0 +1,40 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {featureFlagsTest} from '../../fixtures/featureFlagsTest';
+import {loginTest} from '../../fixtures/loginTest';
+import {dataSetsPageTest} from './fixtures/dataSetsPageTest';
+
+export const test = mergeTests(
+	dataSetsPageTest,
+	featureFlagsTest({
+		'LPS-164563': true,
+	}),
+	loginTest
+);
+
+test('Assert table column labels', async ({dataSetsPage, page}) => {
+	await dataSetsPage.goto();
+	await dataSetsPage.createDataSet();
+
+	const tableColumnLabels = await page
+		.locator('.dnd-thead > .dnd-tr')
+		.first()
+		.locator('.dnd-th')
+		.allInnerTexts();
+
+	const expectedLabels = [
+		'Name',
+		'REST Application',
+		'REST Schema',
+		'REST Endpoint',
+		'Modified Date',
+		'',
+	];
+
+	await expect(tableColumnLabels).toEqual(expectedLabels);
```

Comment:
@kresimir-coko Nice! Much better than six checks I mentioned. But, we're still missing the count check 
```js
await expect(tableColumnsCount).toBe(6);`
``` 
That check _in addition_ to 
```js
await expect(tableColumnLabels).toEqual(expectedLabels);`
```
assures that no other column exists (`Views`).